src/main/java/org/opensearch/neuralsearch/query/NeuralQueryBuilder.java

```diff
@@ -263,7 +310,17 @@ protected QueryBuilder doRewrite(QueryRewriteContext queryRewriteContext) {
                 actionListener.onResponse(null);
             }, actionListener::onFailure)))
         );
-        return new NeuralQueryBuilder(fieldName(), queryText(), queryImage(), modelId(), k(), vectorSetOnce::get, filter());
+        return new NeuralQueryBuilder(
+            fieldName(),
+            queryText(),
+            queryImage(),
+            modelId(),
+            k(),
+            max_distance(),
```

Comment:
getter methods are probably generated, hope it will change once you update the variable names